### PR TITLE
Githook highlighter

### PR DESCRIPTION
**Pull request by Konctantin**
Shell code highlighter for GitHook editor
image
image

### CODE REVIEW

**Review by zeripath — APPROVED** [review]

**Review by sapk — APPROVED** [review]

**Comment by zeripath** [comment]
Sigh... TestGitHubDownloadRepo is failing because someone touched one of the issues it looks at.

**Comment by zeripath** [comment]
@Konctantin - I think we're gonna merge #8973 before this one which will almost certainly conflict with this. Sorry about this.

**Comment by Konctantin** [comment]
Ok, then after merge #8973 i will fix the conflict

**Comment by zeripath** [comment]
Heya! Yeah it did conflict but if you can fix we should be able to get this merged this evening

File: web_src/js/index.js [hunk 1]
```diff
@@ -1486,855 +1486,30 @@ function initOrganization() {
 }
 
 function initUserSettings() {
-    // Options
```
Comment (lunny) [inline web_src/js/index.js]: Why deleted these functions?

**Review by lunny — CHANGES_REQUESTED** [review]

File: web_src/js/index.js (hunk 1 above)
Comment (Konctantin) [reply web_src/js/index.js]: But I didn’t delete anything

**Review by lunny — APPROVED** [review]

**Comment by Konctantin** [comment]
Thanks

**Comment by sapk** [comment]
@Konctantin Thank you for your first contribution.

**Pull request merged**